### DISCUSSION TITLE
more descriptive page titles

### DISCUSSION THREAD

**Discussion by omniraptorr** [pdelfan/ouranos#6] [Ideas]
Please make the html page title show the type and author of the page being viewed.

**Comment by pdelfan** [comment]
Hey @omniraptorr, thanks for bringing this up. I made a few changes to make titles more contextual and descriptive ([see the changes](https://github.com/pdelfan/ouranos/commit/a48ca76ec835ce71b0076aba6f12e82953301de2)). I hope that helps!
